### ISSUE TITLE
Complied release does not run on iOS < 14.0

### ISSUE THREAD

**Issue by manatails** [Skittyblock/SixLS#27]
The .deb release I downloaded from your official repo does not run on devices running on iOS versions earlier than 14.0.

I presume that it was compiled with Xcode 12, which currently has the following issue: [https://github.com/theos/theos/issues/563](url)
I had to recompile the project with Xcode 11.7 in order to make it work under iOS 13. It works without any problems once recompiled.

**Comment by Skittyblock** [comment]
It runs on devices lower than iOS 14 fine (I tested myself), but not on arm64e devices. It says that in the depiction. Even when compiled with Xcode 11, the Swift portions of the code (the entire UI) won't work on arm64e due to an issue I can't fix. I know people that are working on it though, so I will update it as soon as it's fixed.

**Comment by manatails** [comment]
I built the project using with only the changes I PR'd using Xcode 11.7 and iOS 13.5 SDK and it works flawlessly on my arm64e iPad Pro 3rd gen running iOS 13.5. I also uploaded the build on my forked repo so perhaps you could check it out?

**Comment by Skittyblock** [comment]
Correct. It'll work fine on iOS 13. However, it won't work on iOS 14. Either iOS 14 works and 13 doesn't, or 13 does and 14 doesn't. I chose to go with arm64e iOS 14 support since more people are now on iOS 14.
